XBeeLibrary.Xamarin/Connection/Bluetooth/BluetoothInterface.cs

```diff
@@ -185,10 +185,11 @@ public void Open()
 					var retries = BT_CONNECT_RETRIES;
 					while (!IsOpen && retries > 0)
 					{
+						var parameters = new ConnectParameters(forceBleTransport: true);
```

Comment:
Add a little comment explaining the reason of adding the `forceBleTransport` parameter. It will help us to track other possible issues related to the connection in the future.